pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -527,9 +527,12 @@ private boolean processStreamEvents(MessageBatch messagesAndOffsets, long idlePi
     int indexedMessageCount = 0;
     int streamMessageCount = 0;
     boolean canTakeMore = true;
+    boolean noTransformedRows = true;
```

Comment:
rename this to hasTransformedRows = false;

Reply:
Done